packages/synapse-react-client/src/components/DataGrid/SynapseGrid.tsx

```diff
@@ -422,9 +446,9 @@ const SynapseGrid = forwardRef<SynapseGridHandle, SynapseGridProps>(
                     overflowY: 'auto',
                   }}
                 >
-                  <h3>Model snapshot</h3>
-                  {modelSnapshot ? (
-                    <ComplexJSONRenderer value={modelSnapshot} />
+                  <h3>Model</h3>
+                  {model ? (
+                    <ClickableJsonCrdt model={model} />
```

Comment:
The author of json-joy also has this `clickable-json` package which provides this utility, which also displays the CRDT ID of each node in the model:

<img width="356" height="112" alt="Image" src="https://github.com/user-attachments/assets/128daf40-692b-4cb9-83b8-636c8230370a" />

This was useful to validate that the selection object was correct.